client.py

```diff
@@ -193,8 +193,6 @@
 	ewcfg.cmd_exchangerate: ewmarket.rate,
 	ewcfg.cmd_exchangerate_alt1: ewmarket.rate,
 	ewcfg.cmd_exchangerate_alt2: ewmarket.rate,
-	ewcfg.cmd_exchangerate_alt3: ewmarket.rate,
```

Comment:
you basically undid a bunch of my minor patches from me. I'll mark the spots you should change back, starting here